staging/kustomization.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: strapiv4-staging
 bases:
+- ../workload
```

Comment:
Does the workload has the same variables for prod and staging?

Reply:
The difference are in the [secret](https://github.com/dev-launchers/k8s-infra/tree/main/tenants/staging-knative/strapiv4/secrets) and [configmap](https://github.com/dev-launchers/strapiv4/blob/main/staging/configmap.yaml).